### PR TITLE
[EN] week02 corrections

### PR DESCRIPTION
**Pull request by marcozullich**
fixed typos, indenting, em-dash (- → --), missing math env...

### CODE REVIEW

**Comment by marcozullich** [comment]
BTW, 404 🎉 🎉

**Comment by marcozullich** [comment]
Sure I can do that :) 
What kind of work do you need me to do?
a) Just write on the language channels that I did some changes and let the guys there do the rest?
b) Correct the English text commented in the national translations?

File: docs/en/week02/02-1.md [hunk 1]
```diff
@@ -71,11 +71,15 @@ Loss function is a function that is minimized during training. There are two typ
...
 2) Average Loss -
 
-​	For any set of Samples $$S = \{(x[p],y[p]) \mid p=0,1...P-1 \}$$
+​	For any set of Samples 
+$$
+  S = \{(x[p],y[p]) \mid p=0,1...P-1 \}
+$$
 
-​	Average Loss over the Set S is given by :  $$L(S,w) = \frac{1}{P} \sum_{(x,y)} L(x,y,w)$$
+​	Average Loss over the Set $S$ is given by :  $$L(S,w) = \frac{1}{P} \sum_{(x,y)} L(x,y,w)$$
```
Comment (Atcold) [inline docs/en/week02/02-1.md:82]: Like this kind of edits is needed for the [`ES`](https://github.com/Atcold/pytorch-Deep-Learning/blame/b8231a30636b9552ae1980a9be6f9c3de967eb14/docs/es/week02/02-1.md#L146), and [`ZH`](https://github.com/Atcold/pytorch-Deep-Learning/blame/b8231a30636b9552ae1980a9be6f9c3de967eb14/docs/zh/week02/02-1.md#L113) versions.

File: docs/en/week02/02-1.md [hunk 2]
```diff
@@ -97,12 +101,12 @@ A **Gradient Based Method** is a method/algorithm that finds the minima of a fun
   $$
 
 - For SGD (Stochastic Gradient  Descent), the update rule becomes :
-  - Pick a $p$ in $\text{0...P-1}$, then update
+  - Pick a $p$ in $\text{0,\dots,P-1}$, then update
```
Comment (Atcold) [inline docs/en/week02/02-1.md:104]: Same for this one, for [`ZH`](https://github.com/Atcold/pytorch-Deep-Learning/blame/b8231a30636b9552ae1980a9be6f9c3de967eb14/docs/zh/week02/02-1.md#L152), [`ES`](https://github.com/Atcold/pytorch-Deep-Learning/blame/b8231a30636b9552ae1980a9be6f9c3de967eb14/docs/es/week02/02-1.md#L144), [`KO`](https://github.com/Atcold/pytorch-Deep-Learning/blame/b8231a30636b9552ae1980a9be6f9c3de967eb14/docs/ko/week02/02-1.md#L121), and [`IT`](https://github.com/Atcold/pytorch-Deep-Learning/blame/master/docs/it/week02/02-1.md#L83).

**Comment by Atcold** [comment]
Does it make sense? Math corrections should be propagated to all languages.
Otherwise they will diverge.
I think we should have used something similar to how we have handles the figures.

@ebetica, any thoughts on this?

**Comment by marcozullich** [comment]
> 
> 
> Does it make sense? Math corrections should be propagated to all languages.
> Otherwise they will diverge.
> I think we should have used something similar to how we have handles the figures.
> 
> @ebetica, any thoughts on this?

Sure, it makes a lot of sense. We could do something like an Issue for the figures discrepancy I found for one of the past lectures. Just let me know how to proceed :)

**Comment by Atcold** [comment]
For this PR, let's just fix manually all the languages.
Then we can open a new issue, where we talk about the math.
How does it sound?

**Comment by marcozullich** [comment]
OK, fine. I'll get to work now. Sorry for the delay, I've just returned from holidays :)

**Comment by Atcold** [comment]
@marcozullich, I'm confused. Did you push any update to this branch? I don't see anything new here.

**Comment by marcozullich** [comment]
Not yet, I caught a flu last week and I’m still recovering… Da: Alfredo CanzianiInviato: sabato 8 agosto 2020 21:27A: Atcold/pytorch-Deep-LearningCc: Marco Zullich; MentionOggetto: Re: [Atcold/pytorch-Deep-Learning] [EN] week02 corrections (#404) @marcozullich, I'm confused. Did you push any update to this branch? I don't see anything new here.—You are receiving this because you were mentioned.Reply to this email directly, view it on GitHub, or unsubscribe.

**Comment by marcozullich** [comment]
OK, propagated math corrections to the other languages and fixed conflicts with my previous IT week02 review -- hopefully it should be alright now 🙏

**Comment by ebetica** [comment]
> Does it make sense? Math corrections should be propagated to all languages.
> Otherwise they will diverge.
> I think we should have used something similar to how we have handles the figures.
> 
> @ebetica, any thoughts on this?

I think it'll be too much work... You'd have to go and pull out every equation, give it a name, and probably just create a latex macro for it to serve to all languages. Plus - it'd make the source somewhat hard to read.

I maybe support it for long equations and ubiquitous ones, but definitely not stuff like the edits in this PR. I think it's easy enough to just propagate all the changes manually as we find mistakes.

**Pull request merged**